servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ExportRepository.java

```diff
@@ -95,14 +97,37 @@ enum Format {
       })
   private boolean fullScan;
 
+  @CommandLine.Option(
+      names = {"--only-contents-from-branch"},
```

Comment:
Not sure the option name is understandable. Maybe: `--single-branch-current-content`

Reply:
renamed and moved to constant